_config.yml

```diff
@@ -27,6 +27,10 @@ krishna:
     name: Kumar Krishna Agrawal
     email: kumarkagrawal@gmail.com
     web: http://twitter.com/kumarkagrawal
+james:
+    name: James Allingham
+    email: 
+    web: https://twitter.com/JamesAllingham
 
```

Comment:
I wasn't sure if I should add my details here but it seemed to work...

Reply:
Cool, go for it, haha.